### PR TITLE
Updates for 20.11

### PR DESCRIPTION
**Pull request by labkey-tchad**

### CODE REVIEW

File: module.properties [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+# Copyright � 2017-2019 Harvard Pilgrim Health Care Institute (HPHCI) and its Contributors. Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
 # The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-# Funding Source: Food and Drug Administration (�Funding Agency�) effective 18 September 2014 as Contract no. HHSF22320140030I/HHSF22301006T (the �Prime Contract�).
+# Funding Source: Food and Drug Administration (�Funding Agency�) effective 18 September 2014 as Contract no. HHSF22320140030I/HHSF22301006T (the �Prime Contract�).
```
Comment (labkey-adam) [inline module.properties]: Same here

File: module.properties [hunk 2]
```diff
@@ -1,16 +1,16 @@
 #
-# Copyright � 2017-2019 Harvard Pilgrim Health Care Institute (HPHCI) and its Contributors. Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# Copyright � 2017-2019 Harvard Pilgrim Health Care Institute (HPHCI) and its Contributors. Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```
Comment (labkey-adam) [inline module.properties]: Is there a reason this character changed? UTF-8 problem?

File: module.properties [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 #
 #
 
 ModuleClass: com.hphc.mystudies.FdahpUserRegWSModule
-Label: FIXME! Capitalized Short Module Summary
-Description: FIXME! Write a paragraph describing the functionality and the \
-    purpose of this module. The description should contain multiple scentences.
+Label: MyStudies Registration Module
+Description: Primary module for MyStudies Registration server
 URL: http://example.com/FIXME
```
Comment (labkey-adam) [inline module.properties]:
```suggestion
URL: https://github.com/FDA-MyStudies/UserReg-WS
```

File: module.properties [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 License: Apache 2.0
```
Comment (labkey-adam) [inline module.properties]:
```suggestion
License: HPHCI Open Source License
```

File: module.properties [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
 LicenseURL: http://www.apache.org/licenses/LICENSE-2.0
```
Comment (labkey-adam) [inline module.properties]:
```suggestion
LicenseURL: https://github.com/FDA-MyStudies/UserReg-WS/blob/develop/module.properties
```

**Review by labkey-adam — CHANGES_REQUESTED** [review]
See a few minor suggestions...

File: module.properties (hunk 5 above)
Comment (labkey-tchad) [reply module.properties]: I added a separate `LICENCE` file

**Review by labkey-adam — APPROVED** [review]

**Review by naveenr-btc — APPROVED** [review]

**Review by rohitn-boston — APPROVED** [review]

**Pull request merged**